index.js

```diff
@@ -196,7 +196,7 @@ function loadFile(name, dir, options, files) {
   var buffer = fs.readFileSync(filename)
 
   obj.cacheControl = options.cacheControl
-  obj.maxAge = obj.maxAge ? obj.maxAge : options.maxAge || 0
+  obj.maxAge = (typeof obj.maxAge === 'number' && obj.maxAge === obj.maxAge) ? obj.maxAge : options.maxAge || 0
```

Comment:
change this line to `(typeof obj.maxAge === 'number' ? obj.maxAge : options.maxAge) || 0` is better.

also add a test case for this change btw.

Reply:
ok, I've changed the code.